<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBasicWindow</class>
 <widget class="QMainWindow" name="GBSBasicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCCCCC; border: none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #FFFFFF; border: none;</string>
   </property>
   <widget class="QFrame" name="naviFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2667FE; border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnLiveBroadcast">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>130</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>主播直播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBWBroadcast">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>170</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>BW直播</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="danmaFrame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>401</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSummerize">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>汇总</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDanmaku">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>弹幕</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIlleaglWord">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>违规提醒</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPromotor">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>提词器</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="liveFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>571</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头驱动</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>麦克风驱动</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiPreview">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>571</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #424242; border: none;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>开始直播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: #CCCCCC; border: none;</string>
     </property>
     <property name="text">
      <string>开始拉流</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
